components/index.tsx

```diff
@@ -27,11 +27,11 @@ export function TextInput (props: DefaultTextInput['props']): JSX.Element {
   return <DefaultTextInput style={[Default.text, style]} {...otherProps} />
 }
 
-export function NumberText (props: { value: string, decimal?: number }): JSX.Element {
+export function NumberText (props: { value: string, decimal?: number, style?: StyleProp<TextStyle> }): JSX.Element {
```

Comment:
Allow styling the child <Text>

Reply:
We need to deprecate the use of this component entirely, if anyone wants to use NumberFormat to declare it instead and so that we can use tailwind for styling.